### ISSUE TITLE
Can't resolve 'react-select'`

### ISSUE THREAD

**Issue by chulanovskyi** [JedWatson/react-select#3583]
# Are you reporting a bug or runtime error?

After upgrading to version 3.0.1 I've started to receive next error:
`Module not found: Error: Can't resolve 'react-select'`
What I've tried:
* re-install *node_modules* (also deleting the _yarn.lock_) 
* looked for any *breaking changes* section in the readme (the only unclear clause was about "UMD builds deprecation" - could you please explain a little more about this? thanks)
* downgrading to *2.4.3* solved the problem

Have you changed any usage flow? What 'm doing is:
`import Select, { components } from 'react-select';`

Don't know, if it matters, but I'm using Storybook with a webpack full-control mode ([the config can be found here](https://jsbin.com/jorizin/edit?js))

**Comment by danielsogl** [comment]
Same problem for me. This happens because the exported module is wrong:

```js
'use strict';

let unregister = require('/Users/clee/Thinkmill/projects/react-select/node_modules/preconstruct/lib/hook.js').___internalHook('/Users/clee/Thinkmill/projects/react-select');

module.exports = require('/Users/clee/Thinkmill/projects/react-select/packages/react-select/src/index.js');

unregister();

```

**Comment by gwyneplaine** [comment]
thanks @chulanovskyi , I'm looking into it now

**Comment by ezzkht** [comment]
same here. "Module not found: Can't resolve 'react-select' in ...."

**Comment by gwyneplaine** [comment]
@chulanovskyi  @danielsogl  @ezzkht  this should now resolved in v3.0.3. 
Thanks for the quick report on the issue

**Comment by chulanovskyi** [comment]
> @chulanovskyi @danielsogl @ezzkht this should now resolved in v3.0.3.
> Thanks for the quick report on the issue

Yeap, that works, thanks!
